pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -80,6 +80,7 @@ let
           "--disable-decimal-float" # requires libc
           "--disable-libmpx" # requires libc
           "--disable-hosted-libstdcxx" # requires libc
+          "--disable-hosted-libstdcxx"
```

Comment:
Isn't this identical to the line above?

Reply:
Ah my bad, this probably happened during a rebase